### PR TITLE
chore: use eslint-config-oss

### PR DESCRIPTION
**Pull request by fastfrwrd**
## Hey, I just made a Pull Request!

Uses the newly extracted eslint-config-oss in place of the custom notice config.

https://github.com/spotify/eslint-config-oss

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -1,15 +1,3 @@
...
-  rules: {
-    ...base.rules,
-    'notice/notice': [
-      'error',
-      {
-        templateFile: path.resolve(__dirname, 'scripts/copyright.js'),
-      },
-    ],
-  },
+  extends: ['./packages/cli/config/eslint.js', '@spotify/eslint-config-oss'],
```
Comment (Rugvip) [inline .eslintrc.js:2]: Can this still be `@backstage/cli/config/eslint`? Better as a reference for standalone apps that don't have the local cli package.

File: scripts/copyright.js [hunk 2]
```diff
@@ -1,15 +0,0 @@
-/*
- * Copyright <%= YEAR %> Spotify AB
```
Comment (Rugvip) [inline scripts/copyright.js:2]: Noticed you're not using `<%= YEAR %>` in `@spotify/eslint-config-oss`, is that a deliberate choice?

**Comment by Rugvip** [comment]
Thanks! Good chore work :grin:

File: scripts/copyright.js (hunk 2 above)
Comment (fastfrwrd) [reply scripts/copyright.js:2]: Argh it most certainly is not! I know exactly what happened there and will release a patch fix.

File: .eslintrc.js (hunk 1 above)
Comment (fastfrwrd) [reply .eslintrc.js]: agreed - I didn't think of the best way to do that at first, but I just pushed an update I think satisfies you while also avoiding a lot of deeply-nested spreads and leans more comfortably on the `extend` feature. lmk if that works!

Comment (fastfrwrd) [reply .eslintrc.js]: tl;dr: `require.resolve('@backstage/cli/config/eslint.js')`

**Review by freben — APPROVED** [review]
Neat, thanks!

**Review by stefanalund — APPROVED** [review]

**Pull request merged**